app/serializers/spotlight/exhibit_export_serializer.rb

```diff
@@ -34,6 +34,10 @@ class ExhibitExportSerializer < Roar::Decorator
       property prop
     end
 
+    property :theme, if: lambda { |options|
+      Spotlight::Engine.config.exhibit_themes.include? options[:doc]['theme']
```

Comment:
```suggestion
      Spotlight::Engine.config.exhibit_themes.include? options[:input]
```

This is just a guess from skimming through the doc